data/json/items/gun/45.json

```diff
@@ -271,5 +271,42 @@
     "built_in_mods": [ "match_trigger" ],
     "default_mods": [ "suppressor" ],
     "magazines": [ [ "45", [ "usp45mag" ] ] ]
+  },
+  {
+    "id": "walther_ppq_45",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "Walther PPQ .45 ACP",
+    "description": "The Walther PPQ is a semi automatic pistol designed as a potential replacement for the Walther P99.",
+    "weight": 714,
+    "volume": "120 ml",
```

Comment:
120 mL is a bit too small.  Considering the ppq as a right triangular prism, with dimensions of 188mm in length, 147mm in height, and 34mm in width (from wikipedia from the .45ACP model). The volume of a right triangular prism is base*height, or (1/2L*W) *H.  So our PPQ in .45 has a volume of 469812 cubic mm, or ~470mL.